### ISSUE TITLE
What are available options & methods that can be used?

### ISSUE THREAD

**Issue by hozefaj** [sdeering/jquery-quick-pagination#2]
Could you provide all the options & methods that can be used. Also what the options stand for. Need to improve documentation around the utility.

**Comment by sdeering** [comment]
@hozefaj  I agree with you the documentation needs improving! Is this something you might be able to help me with? If so I can add you to the list of contributors for the plugin.

**Comment by hozefaj** [comment]
@sdeering Yes I can help you with this.

**Comment by sdeering** [comment]
@hozefaj thanks!
